autobahn_sync/core.py

```diff
@@ -72,10 +73,11 @@ def bootstrap_and_callback():
             self._bootstrap(blocking, url=url, realm=realm, **kwargs)
             if callback:
                 callback()
+            self._callbacks_runner.start()
```

Comment:
Just to be able to call the callback if blocking=True. Since the blocking is done in the callback_runner